app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h4>Here is single of user.</h4>
\ No newline at end of file
+<div class = "container">
+    <div class = "user-row">
+        <div class = "photo">
+            <%= image_tag @user.photo %>
+        </div>
+        <div class = "intro-details">
+            <p class = "user-name"><%= @user.name %></p>
+            <p class = "post-counter">Number of posts:<%= @user.posts_counter %></p>
+        </div>
+    </div>
+    <div class = "bio">
+        <p> Bio</p>
+        <p><%= @user.bio %></p>
+    </div>
+    <% @post.each do |p| %>
+        <div class = "post-row">
+            <p class ="post-title"> <%= p.title%></p>
+            <p class ="post-text"><%= p.text%></p>
+            <p class="counters"> Comments: <%= p.comments_counter%>,
+            Likes: <%= p.likes_counter%></P>
```

Comment:
- [ ] Let's remember the model methods we created in the previous milestone. We want to get only the 3 most recent posts for a user.